api/src/upload/upload.py

```
@@ -0,0 +1,182 @@
+from pathlib import Path
+import rasterio
+import hashlib
+import shutil
```

Ruff (0.8.0)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)